charts/coturn/templates/certificate.yaml

```diff
@@ -0,0 +1,55 @@
+{{- if .Values.tls.enabled }}
+{{- if .Values.tls.certManager.enabled }}
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: coturn-auto-certificate
+  namespace: {{ .Release.Namespace }}
+  labels:
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  issuerRef:
+    name: {{ .Values.tls.certManager.issuer.name }}
+    kind: {{ .Values.tls.certManager.issuer.kind }}
+  usages:
+    - server auth
+  duration: 2160h # 90 days, ignored by LE
+  renewBefore: 360h # 15 days
+  isCA: false
+  secretName: coturn-tls
+
+  privateKey:
+    algorithm: ECDSA
+    size: 384
+    encoding: PKCS1
+    rotationPolicy: Always
+
+  {{- if eq (len .Values.tls.certManager.dnsNames) 0 }}
+  {{- fail "List of DNS names for automatically provisioned certificate is empty" }}
+  {{- else }}
+  dnsNames:
+  {{- with .Values.tls.certManager.dnsNames }}
+    - {{ . | quote }}
+  {{- end }}
+  {{- end }}
+{{- else }}
+apiVersion: v1
```

Comment:
Please drop the else branch entirely. If the user doesn't want to use cert-manager, they should refer to a secret that's externally created (and updated).

The current state of things requires people to re-run helm whenever they rotate the cert manually, which is not very practical.

Reply:
Done in [3cae51b](https://github.com/wireapp/wire-server/pull/2620/commits/3cae51b9080c3c316e57fa93fe396418c348f4c9)